### PR TITLE
docs: Document our loadingTriangle component

### PR DESCRIPTION
**Pull request by ryan953**
Docs for the loadingTriangle

SCR-20230918-pmnz

It's leveraging this "NegativeSpaceContainer" which is extracted out of the replayPlayer, to create the SizingWindow helper. That helper is really important because it show the transparent parts in both light & dark themes.
The extracted component is also documented, because that's why we're here! document all the things!
SCR-20230918-pmmu

### CODE REVIEW

File: static/app/components/replays/replayPlayer.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import LoadingIndicator from 'sentry/components/loadingIndicator';
 import BufferingOverlay from 'sentry/components/replays/player/bufferingOverlay';
 import FastForwardBadge from 'sentry/components/replays/player/fastForwardBadge';
 import {useReplayContext} from 'sentry/components/replays/replayContext';
+import SizingWindow from 'sentry/components/stories/sizingWindow';
```
Comment (evanpurkhiser) [inline static/app/components/replays/replayPlayer.tsx:9]: Unrelated?

Comment (evanpurkhiser) [reply static/app/components/replays/replayPlayer.tsx:9]: oh hmmm

File: static/app/components/stories/sizingWindow.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import styled from '@emotion/styled';
```
Comment (malwilley) [inline static/app/components/stories/sizingWindow.tsx:1]: Since this is used in replays and in stories, shouldn't this be in the base `/components` directory?

**Review by malwilley — APPROVED** [review]
